source/PlayerInfo.cpp

```diff
@@ -683,78 +683,45 @@ const Date &PlayerInfo::GetDate() const
 	return date;
 }
 
-
-
-// Set the date to the next day, and perform all daily actions.
-void PlayerInfo::IncrementDate()
+// Set the date, and perform all daily actions the given number of times.
+// Supports zero negative values.
```

Comment:
```suggestion
// Supports null and negative values.
```
then again 0 does nothing so it doesnt make sense but whatever

Reply:
Yeah, maybe I shouldn't point out 0. I'd rather avoid saying null, because people might mistake that for a nullptr (even though these are not pointers).

Reply:
right, well write what you wish, it was mostly missing a connector between 0 and negative
we could just say negative values